### PR TITLE
Add rtbd gamespecs

### PR DESCRIPTION
**Pull request by PriyankaVaidya**

### CODE REVIEW

File: statistics/read-rtdb-gamespecs.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+});
+
+const db = admin.database();
+const dbRef = db.ref("/gameBuilder");
+const dbTarget = db.ref("/gamePortal/gamesInfoAndSpec/gameInfo");
+const dbTargetSpec = db.ref("/gamePortal/gamesInfoAndSpec/gameSpecsForPortal");
+var dbTargetElement;
+
+
+var writeUserData = function (promisesSpecs, promisesElemes, promisesImages,itemsElems,itemsSpecs,dbTargetElem) {
```
Comment (apsdehal) [inline statistics/read-rtdb-gamespecs.js:17]: You don't need to pass most of the arguments here now. Remove the empty arrays that are passed and create them locally.

File: statistics/read-rtdb-gamespecs.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+//console.log(dbTargetElem);
+  Promise.all(promisesImages).then(
+    (promises) => {
+      promises = promises.filter(promise => !(promise instanceof Error));
+      
+      errors = promises.filter(promise => (promise instanceof Error));
```
Comment (apsdehal) [inline statistics/read-rtdb-gamespecs.js:23]: Use const here.

File: statistics/read-rtdb-gamespecs.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    return Promise.all(promisesSpecs).then(
+      (promises) => {
+        promises = promises.filter(promise => !(promise instanceof Error));
+        
+        errors = promises.filter(promise => (promise instanceof Error));
+        console.log("Errors Encountered:" + errors);
+        console.log(" Finished Writing Specs");
+      }
+    )
+}).then(finalVal => {
```
Comment (apsdehal) [inline statistics/read-rtdb-gamespecs.js:60]: Where are you doing `admin.app().delete`? Weren't you supposed to do it here?

File: statistics/read-rtdb.js [hunk 4]
```diff
@@ -1,15 +1,88 @@
-const serviceAccount = require("../../universalgamemaker-firebase-adminsdk.json");
+//const serviceAccount = require("../../universalgamemaker-firebase-adminsdk.json");
```
Comment (apsdehal) [inline statistics/read-rtdb.js:1]: Reset this file or remove changes here.

This shouldn't be visible in this PR.

**Review by apsdehal — CHANGES_REQUESTED** [review]

**Comment by PriyankaVaidya** [comment]
Closing this PR post discussion with Professor

**Pull request closed without merging**